collect_app/src/main/java/org/odk/collect/android/widgets/WidgetFactory.java

```diff
@@ -205,13 +205,13 @@ public QuestionWidget createWidgetFromPrompt(FormEntryPrompt prompt) {
                 // This means normal appearances should be put before search().
```

Comment:
This comment should be removed. The documentation at https://xlsform.org/en/#dynamic-selects-from-pre-loaded-data also states that the `search()` appearance needs to come last and that can be updated.

Reply:
Done.